sdk/anomalydetector/ai-anomaly-detector/review/ai-anomaly-detector.api.md

```diff
@@ -0,0 +1,120 @@
+## API Report File for "@azure/ai-anomaly-detector"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import * as coreHttp from '@azure/core-http';
+import { KeyCredential } from '@azure/core-auth';
+import { OperationOptions } from '@azure/core-http';
+import { PipelineOptions } from '@azure/core-http';
+import { TokenCredential } from '@azure/identity';
+
+// @public
+export class AnomalyDetectorClient {
+    constructor(endpointUrl: string, credential: TokenCredential | KeyCredential, options?: AnomalyDetectorClientOptions);
+    detectChangePoint(body: DetectChangePointRequest, options?: OperationOptions): Promise<AnomalyDetectorClientDetectChangePointResponse>;
+    detectEntireSeries(body: DetectRequest, options?: OperationOptions): Promise<AnomalyDetectorClientDetectEntireResponse>;
+    detectLastPoint(body: DetectRequest, options?: OperationOptions): Promise<AnomalyDetectorClientDetectLastPointResponse>;
+    readonly endpointUrl: string;
```

Comment:
is it worthwhile to expose this? I have no strong opinion, just not sure why someone would need it if they already have to feed it into the constructor.